src/Components/Shifting/BadgesList.tsx

```diff
@@ -91,17 +52,31 @@ export default function BadgesList(props: any) {
         ...dateRange(t("modified"), "modified_date"),
         badge(t("disease_status"), "disease_status"),
         badge(t("breathlessness_level"), "breathlessness_level"),
-        value(t("assigned_to"), "assigned_to", assignedUsername),
+        value(
+          t("assigned_to"),
+          "assigned_to",
+          qParams.assigned_to
+            ? `${assignedUser?.results[0].first_name || ""} ${
+                assignedUser?.results[0].last_name || ""
+              }`
+            : ""
```

Comment:
refactor this logic out to a reusable function `formatFacilityBadgeValue`, since it's used in multiple badges?

Reply:
```
export function formatFullName(user: UserModel | undefined): string {
  if (user) {
    return `${user.first_name || ""} ${user.last_name || ""}`;
  } else {
    return "";
  }
}
```

any changes required for this function